src/app/projects/projects-list/projects-list.component.html

```diff
@@ -122,6 +124,11 @@
             <p *ngIf="!project.technologies.length">-</p>
           </ul>
         </td>
+
+        <td class="vf-table__cell">
+          {{project.cellCount}}<p *ngIf="project.cellCount === undefined || project.cellCount === null">-</p>
```

Comment:
seems to me cleaner
```suggestion
          <p *ngIf="project.cellCount; else notProvided">{{project.cellCount}}</p>
          <ng-template #notProvided>-</ng-template>
```

Reply:
but this would also display "0-" when cell count is 0

Reply:
ah no, you're right

Reply:
🥳  thanks! i like that

Reply:
You could also just do `{{project.cellCount ?? '-'}}` right? `??` syntax is like `||` but treats 0 as truthy

Reply:
?? doesn't seem to work but `{{project.cellCount || '-' }}` does. I'm using that. Thanks for the suggestions @ke4 and @jacobwindsor !!!

Reply:
hold on, I want to also use the decimal pipe, I'm changing it to 

`{{project.cellCount ? (project.cellCount | number) : '-' }}`

Reply:
This was the error when using ??

```
 "Parser Error: Conditional expression project.cellCount ?? '-' requires all 3 expressions at the end of the expression [{{project.cellCount ?? '-' }}] in projects-index/src/app/projects/projects-list/projects-list.component.html@128:13"
```

Reply:
@jacobwindsor It is sadly not supported in templates yet: https://github.com/angular/angular/issues/36528